options/locale/locale_en-US.ini

```diff
@@ -2371,6 +2371,7 @@ team_desc_helper = Describe the purpose or role of the team.
 team_access_desc = Repository access
 team_permission_desc = Permission
 team_unit_desc = Allow Access to Repository Sections
+team_unit_helper = The following access control policy has no effect on the public repositories in this organization.
```

Comment:
I don't think it's "has no effect". Although I can't set up a team with `No Access` for public repositories because "public" means everyone can `Read`, I can still set up a team with `Write` to decide who can write the public repositories.

Reply:
Maybe we can append some words to the tooltip of `No Access`: "It has no effect for public repositories."
<img width="1378" alt="image" src="https://user-images.githubusercontent.com/9418365/215414289-a3d0ed74-5f5a-46b5-9ecc-d4def79bd4bd.png">